frontend/public/components/container-selector.tsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-unused-vars, no-undef */
+// import * as _ from 'lodash-es';
+import * as React from 'react';
+import * as _ from 'lodash-es';
+
+import { FormGroup, Checkbox, Label } from 'patternfly-react';
```

Comment:
nit: alphabetize imports

patternfly-react imports should go with the other yarn dependencies in the grouping above

Reply:
Are these PF3 components? We should be using PF4